master/docs/relnotes/0.9.0.rst

```diff
@@ -0,0 +1,625 @@
+Release Notes for Buildbot ``0.9.0``
+========================================
+
+.. warning::
+
+    The following are temporary release notes for unreleased version of buildbot.
+
+The following are the release notes for Buildbot ``0.9.0``.
+
+This is a concatenation of important changes done between 0.8.12 and 0.9.0.
+This does not contain details of the bug fixes related to the nine beta and rc period.
+
+See :ref:`Upgrading to Nine` for a guide to upgrading from 0.8.x to 0.9.x
+
+Master
+------
+
+This version represents a refactoring of Buildbot into a consistent, well-defined application composed of loosely coupled components.
+The components are linked by a common database backend and a messaging system.
+This allows components to be distributed across multiple build masters.
+It also allows the rendering of complex web status views to be performed in the browser, rather than on the buildmasters.
+
+The branch looks forward to committing to long-term API compatibility, but does not reach that goal.
+The Buildbot-0.9.x series of releases will give the new APIs time to "settle in" before we commit to them.
+Commitment will wait for Buildbot-1.0.0 (as per http://semver.org).
+Once Buildbot reaches version 1.0.0, upgrades will become much easier for users.
+
+To encourage contributions from a wider field of developers, the web application is designed to look like a normal AngularJS application.
+Developers familiar with AngularJS, but not with Python, should be able to start hacking on the web application quickly.
+The web application is "pluggable", so users who develop their own status displays can package those separately from Buildbot itself.
+
+Other goals:
+
+ * An approachable HTTP REST API, with real time event features used by the web application but available for any other purpose.
+ * A high degree of coverage by reliable, easily-modified tests.
+ * "Interlocking" tests to guarantee compatibility.
+   For example, the real and fake DB implementations must both pass the same suite of tests.
+   Then no unseen difference between the fake and real implementations can mask errors that will occur in production.
+
+Requirements
+~~~~~~~~~~~~
+
+The ``buildbot`` package requires Python 2.6 or higher -- Python 2.5 is no longer supported.
```

Comment:
yeah. I'd appreciate any suggestionor help to reduce the amount of text here.
I think there is a lot of TLDR issue in this document.